translations/cy/ccs_household_gb_wls_cy.po

```diff
@@ -1,3397 +1,2614 @@
-# Translations template for PROJECT.
-# Copyright (C) 2020 ORGANIZATION
-# This file is distributed under the same license as the PROJECT project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
-#
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: PROJECT VERSION\n"
+"Project-Id-Version: eq-census\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2020-07-22 16:11+0100\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"POT-Creation-Date: 2020-12-07 09:43+0000\n"
+"PO-Revision-Date: 2020-12-16 07:44\n"
+"Last-Translator: \n"
```

Comment:
Should this be populated?

Reply:
It's part of our pot files now, so it's what babel 2.9.0 introduced probably..

Reply:
This is how it came down, so I'm not sure what value should be there?